### PR TITLE
Added Changelog in website

### LINKED ISSUE #2973: Update the link to gh pages of Phimp.me in the read me.
**Describe the chore**
In the readme, the link to the gh pages of phimpme is given as "https://phimp.me". But the link is actually "fossasia.github.io/phimpme-android". This should be corrected in the gh pages section under branch policy in the readme. 

**Would you like to work on the issue?**

- [ ] Yes
- [X] No
- Other:

### PR DESCRIPTION
**Pull request by Sid911**
Fixed #2973 #2974 

## Changes: 
- Added https://sphinx-git.readthedocs.io/en/latest/index.html git-sphinx for changeLog
- Added the hosting page link as per #2973 

## Screenshots of the change: 

PREVIEW: https://sid911.github.io/phimpme-android/
image

Don't worry the changelog heading is above the changelog.... its just like that in the preview because its coming from another branch which is old 😅

### CODE REVIEW

**Comment by Sid911** [comment]
image
this built just fine in the pr... but not in my repo 😅

**Comment by Sid911** [comment]
@yashk2000 @arundhatigupta Please review 😇

File: README.md [hunk 1]
```diff
@@ -87,6 +87,7 @@ We have the following branches
 
  * **gh-pages**
    Hosting the landing page [phimp.me](http://phimp.me)
```
Comment (yashk2000) [inline README.md]: Please remove this line entirely. Just keep the new one and remove the word currently from it.

Comment (Sid911) [reply README.md]: ohk... working on it 👍

**Comment by Sid911** [comment]
@yashk2000 Done 😇

**Pull request merged**